Quest tabs for UIV and UMV

srdr2k3 opened the pull request:
UEV Tab (Just the Awakened Core and Infinity gear):
![image](https://github.com/user-attachments/assets/1f852d36-6fb0-47c4-8efd-6a1e3067b58a)

UIV Tab:
![image](https://github.com/user-attachments/assets/232d0738-085b-4b08-b9a9-4e3a2c6a4004)

UMV:
![image](https://github.com/user-attachments/assets/1dad9d1a-a167-449a-bd00-0d0326b0c107)

Endgame Goals:
![image](https://github.com/user-attachments/assets/df3c0533-6b21-4097-8a76-faf3e6aa0b8d)

This PR aims to make room for the content to be added to UIV and UMV tiers and to make new quests for the recently added content. Hopefully, it will also give a sense of progression to the confused people who make it this far.

Hopefully, this doesn't break much. Everything does look fine on my client.
Also I have no idea why the first 2 commits are there lol. I started work on this after those were merged in the main repo and my fork.

EnderProyects commented:
There should be "joints" of requirement between awakened core and the inf armour, idk something like this ( maybe its hidden idk)
![image](https://github.com/user-attachments/assets/1f497cf1-eef4-4e46-8760-54af722d3ede)

chochem commented:
there should not be joints. the problem is more fundamental: those quests are completely in the wrong place

srdr2k3 commented:
> There should be "joints" of requirement between awakened core and the inf armour, idk something like this ( maybe its hidden idk) ![image](https://private-user-images.githubusercontent.com/45178400/352501820-1f497cf1-eef4-4e46-8760-54af722d3ede.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5OTA2MjksIm5iZiI6MTcyMTk5MDMyOSwicGF0aCI6Ii80NTE3ODQwMC8zNTI1MDE4MjAtMWY0OTdjZjEtZWVmNC00ZTQ2LTg3NjAtNTRhZjcyMmQzZWRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDEwMzg0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0NDg0NzEyM2E2ZjRmMTVkZTc3Yzc3MzI0ZTY2MDNlMDk5NDk0ZGUyZjk2OTI1MDg3NTI2ZjZlN2U5YjUxMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3RNSDdJu-ZY1FfUqwVLDnrUZNb7cneUgQYo6SUFieyQ)

This is the shape that endgame goals was planned around, the lines will break it completely.
![Untitled](https://github.com/user-attachments/assets/57751d5b-7842-43cf-94bc-577857b822be)

chochem commented:
and the bigger problem: there is not enough content for 3 tabs. UIV and UMV are not remotely finished tiers and UMV probably wont be for a long time to come. I propose we just close this.

EnderProyects commented:
I like the tabs, is more tier distributed, but is a little odd that there are quests "floating" in endgame goals

srdr2k3 commented:
And I propose this getting finished up and merged into full pack. Content and Gimmicks have been added to the pack, they need documenting.

I can't emphasize enough on the times I was so confused after finishing UEV and had to resort to the discord to have a sense of what I should be doing. UIV and UMV desperately need documentation and space for new quests.

I hope the opinions of a player who has finished the pack will be considered.

chochem commented:
new quests are not just totally fine but actually great.

Ethryan commented on config/betterquesting/DefaultQuests/QuestLines/Tier11UIV-ytf2QCzUSk6ihcYeRbwblw==/QuestLine.json:
```diff
@@ -0,0 +1,19 @@
+  "questLineIDHigh:4": -3830322202564867506,
+  "properties:10": {
+    "betterquesting:10": {
+      "visibility:8": "NORMAL",
```
Set this to “UNLOCKED”

Ethryan commented on config/betterquesting/DefaultQuests/QuestLines/Tier12UMV-l37V2WvbSHGZQ5oMgAfBuw==/QuestLine.json:
```diff
@@ -0,0 +1,19 @@
+  "questLineIDHigh:4": -7530346397122738063,
+  "properties:10": {
+    "betterquesting:10": {
+      "visibility:8": "NORMAL",
```
This too

srdr2k3 commented:
Okay, I thought I'd finish up on the quest since I opened the pack already.

The PR is ready for review and the images have been updated with the final shapes of the quest tabs

chochem reviewed (dismissed):
you didnt split things right. most of these are not uxv+ 
![image](https://github.com/user-attachments/assets/2a5ddca4-b4c9-41bb-938e-d417fe05aeea)

srdr2k3 commented:
Okay, as per discussion in https://discord.com/channels/181078474394566657/949447391587733504/1266471056839807077 Endgame Goals has been stripped of all non-UXV quests. I've moved all of the quests to their respective tabs and tiers.

Updated Images of each tab can be found in the main description.

Ready to review once again.

srdr2k3 commented:
Got to some quick fixes, sorry about that. I'll request for review again when it's done

srdr2k3 commented:
Good to go, sorry about the delay again

NotAPenguin0 commented on config/betterquesting/DefaultQuests/Quests/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==/Eternity-wGq5FvuuROyyMPbT_SkMYg==.json:
```diff
@@ -29,7 +37,7 @@
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Definitely consider this for the better Magnetohydrodynamically Constrained Star Matter recipe.\n\nKickstarting the loop is expensive and RNG based, you need 20,736L of Universium for the first 5 attempts and a catalyst.\n\nThe QFT recipe has a 25% chance of yielding Eternity per attempt, you should probably boost the QFT with the plasmas, and focus Eternity.\n\n[note]Did you kno ...
+      "desc:8": "Definitely consider this for the improved Magnetohydrodynamically Constrained Star Matter recipe. You’ll also need it to make MHDCSM parts.\n\nKickstarting the loop is expensive and RNG-based. You need 20,736 L of Universium for the first 5 attempts and a catalyst.\n\nThe QFT recipe has a 25% chance of yielding Eternity per attempt. You should probably boost the QFT with plasmas  ...
```
Would definitely mention *how* to toggle the QFT fluid mode instead of only mentioning that it exists

srdr2k3 replied:
resolved

NotAPenguin0 commented on config/betterquesting/DefaultQuests/Quests/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==/Magnetohydrodyna-52Yelo6KSLGumzXo-_F5DQ==.json:
```diff
@@ -0,0 +1,52 @@
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Or MHDCSM for short.\n\nThis is your primary metal for UXV components. It’s tricky to make and extremely expensive, as all endgame materials should be.\n\nInstead of solidifying it directly, you need to coat Eternity parts with it.\n\nHave fun!"
```
Is MHDCSM a metal? I'd just say material.

srdr2k3 replied:
resolved

NotAPenguin0 reviewed (changes requested):
Looks good overall, just a couple nitpicks

NotAPenguin0 reviewed (approved):
Content looks good to me
